### PR TITLE
Bananana

### PR DESCRIPTION
**Pull request by Bulldog2740**

### CODE REVIEW

File: src/main/java/fruits/Fruit.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package fruits;
+
+import java.time.LocalDate;
+import org.codehaus.jackson.annotate.JsonIgnore;
```
Comment (Taniann) [inline src/main/java/fruits/Fruit.java]: add blank line between imports and class declaring

File: src/main/java/fruits/Fruit.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        return price;
+    }
+
+    public String getDateofDelivery() {
+        return dateofDelivery;
+    }
+
+    public int getShelfLife() {
+        return shelfLife;
+    }
```
Comment (Taniann) [inline src/main/java/fruits/Fruit.java]: add blank line between methods

File: src/main/java/fruits/Fruit.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+@JsonIgnore
```
Comment (Taniann) [inline src/main/java/fruits/Fruit.java]: add needed white spaces here

File: src/main/java/fruits/Fruit.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+public LocalDate getDateOfSpoil(){
```
Comment (Taniann) [inline src/main/java/fruits/Fruit.java]: formate your code
use "ctrl+alt+l" if you use intelij idea

File: src/main/java/fruits/FruitStore.java [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+        return listOfFruits;
+    }
+
+    private List<Fruit> filterByType(FruitType type, List<Fruit> list) {
+        List<Fruit> listOfFruits = list.stream().filter(fruit -> fruit.getFruitType()
+                .equals(type)).collect(Collectors.toList());
+        return listOfFruits;
+    }
+
+    public List<Fruit> getFruitsStorage() {
```
Comment (Taniann) [inline src/main/java/fruits/FruitStore.java]: firstly public methods, then private
[Ordering of class contents](https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents)

File: src/test/java/fruitstore/FruitTest.java [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+import fruits.FruitType;
+import fruits.Util;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.time.LocalDate;
+
+import static junit.framework.TestCase.assertTrue;
+
+public class FruitTest {
```
Comment (Taniann) [inline src/test/java/fruitstore/FruitTest.java]: add tests for methods getSpoiledFruits(), getAvailableFruits, getAddedFruits()

Comment (Bulldog2740) [reply src/test/java/fruitstore/FruitTest.java]: but i have methods that you say. at FruitStoreTest.java
look at that file 
image
image
image

**Pull request merged**